components/ShareRoom/ShareRoomSidebarList.tsx

```
@@ -36,17 +36,19 @@ export default function ShareRoomSidebarList() {
-						query: { sharedRoomTitle: room.sharedRoomTitle },
-					}}
-					aria-label={`シェアルーム: ${room.sharedRoomTitle}`}
-					className="flex items-center gap-2 rounded-lg px-4 py-4 text-gray-700 hover:bg-gray-100 hover:text-blue-500 transition-colors"
-				>
-					{room.sharedRoomTitle}
-				</Link>
+				<div className="border-b">
+					<Link
+						key={room.shareRoomId}
+						href={{
+							pathname: `/albumShareRoom/${room.shareRoomId}`,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)